app/assets/javascripts/sitewide/form_validators.js.erb

```diff
@@ -138,10 +131,10 @@ function filesSizeValidator(ev, fileInputs, fileTypeEnum) {
     }
     if (file.size > fileTypeEnum) {
       switch (fileTypeEnum) {
-        case FileTypeSizeEnum.FILE:
-         return "<%= I18n.t 'general.file.size_exceeded', file_size: FILE_MAX_SIZE %>".strToErrorFormat();
-        case FileTypeSizeEnum.AVATAR:
-          return "<%= I18n.t 'general.file.size_exceeded', file_size: AVATAR_MAX_SIZE %>".strToErrorFormat();
+        case FileTypeEnum.FILE:
```

Comment:
Ruby preprocesses the file, hardcodinh the values into the file, so this cannot be made dynamically, e.g. without switch statement - Ruby and other (JS, HTML) code cannot interact. Best way to understand this is to try to write more elegantly without the switch (or if) statement, and you'll see.